modules/machineset-migrating-compute-nodes-to-new-sd-rhv.adoc

```diff
@@ -0,0 +1,50 @@
+[id="machineset-migrating-compute-nodes-to-new-sd-rhv"]
+= Migrating compute nodes to a different storage domain in {rh-virtualization}
+
+.Prerequisites
+
+* You are logged into the {rh-virtualization-engine-name}.
+* You have the name of the target storage domain.
+
+.Procedure
+
+. Identify the virtual machine template:
++
+[source,terminal]
+----
+$ oc get -o jsonpath='{.spec.template.spec.providerSpec.value.template_name}{"\n"}' machineset
+----
+
+. Create a new virtual machine in the {rh-virtualization-engine-name}, based on the template you identified. Leave all other settings unchanged. For details, see  link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Creating_a_Virtual_Machine_Based_on_a_Template[Creating a Virtual Machine Based on a Template] in the Red Hat Virtualization _Virtual Machine Management Guide_.
++
+[TIP]
+====
+You do not need to start the new virtual machine.
+====
+
+. Create a new template from the new virtual machine. Specify the target storage domain under *Target*. For details, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Creating_a_template_from_an_existing_virtual_machine[Creating a Template] in the Red Hat Virtualization _Virtual Machine Management Guide_.
+
+. Add a new machine set to the {product-title} cluster with the new template.
+.. Get the details of the current machine set:
++
+[source,terminal]
+----
+$ oc get machineset -o yaml
+----
+.. Use these details to create a machine set. For more information see _Creating a machine set_.
```

Comment:
This should either be a link or contain the word "above" to direct people to the right place.